tests/infra/node.py

```diff
@@ -446,25 +449,29 @@ def is_stopped(self):
     def is_joined(self):
         return self.network_state == NodeNetworkState.joined
 
-    def wait_for_node_to_join(self, timeout=3):
+    def wait_for_node_to_join(self, *args, timeout=3, **kwargs):
         """
         This function can be used to check that a node has successfully
         joined a network and that it is part of the consensus.
         """
-        # Until the node has joined, the SSL handshake will fail as the node
-        # is not yet endorsed by the service certificate
-
-        try:
-            with self.client(connection_timeout=timeout) as nc:
-                rep = nc.get("/node/commit")
-                assert (
-                    rep.status_code == 200
-                ), f"An error occurred after node {self.local_node_id} joined the network: {rep.body}"
-                self.network_state = infra.node.NodeNetworkState.joined
-        except infra.clients.CCFConnectionException as e:
-            raise TimeoutError(
-                f"Node {self.local_node_id} failed to join the network"
-            ) from e
+        start_time = time.time()
```

Comment:
This logic is different now that we check that a node has joined via the node-endorsed RPC interface.